### ISSUE TITLE
Memory Leak? #2 (OSX, C++)

### ISSUE THREAD

**Issue by ryohajika** [alphacep/vosk-api#251]
I use the api with some cpp code upon the openFrameworks library, and it works pretty much fine **for a short while**. but after that I see the software crashes highly likely.
We have another similar issue seems on memory leak as #209, I inspected my app with the Instruments app comes with OSX's Xcode suite.
It turns out, there are some memory leak event at some point in the timeline and it seems them are derived from the kaldi library. I'm just wondering if there is any solution to solve/reduce the problem. Does anybody have an idea on the compatibility between different version of kaldi and vosk-api?

**Comment by nshmyrev** [comment]
Please provide some more precise information - what is the memory usage exactly, how does it grow, what code do you use, can you reproduce the leak with our sample code and so on.

**Comment by ryohajika** [comment]
Hi @nshmyrev ! Thanks for your response and sorry for my slow reply.
I created [a vosk addon for openFrameworks](https://github.com/ryohajika/ofxVosk) and I tried to recreate the issue based on the example project in the repo. Though it seems that the rest of my code which doesn't have any direct relation to either Vosk or Kaldi had an issue.
I want to share the part of the code and discuss where the leak happens but I can't do that at this point... 
Let me close this issue now and I'll reopen this once I found the potential memory leak due to vosk/kaldi again. Thank you :pray:
